package/yast2-bootloader.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-bootloader
-Version:        4.3.25
+Version:        4.3.26
```

Comment:
As said in https://github.com/yast/yast-bootloader/pull/640/files#r616658724, 4.4.0 should be the new version number for master.